src/rdkafka_conf.c

```diff
@@ -470,6 +470,22 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  "protocol. See manual page for `ciphers(1)` and "
 	  "`SSL_CTX_set_cipher_list(3)."
 	},
+#if OPENSSL_VERSION_NUMBER >= 0x1000200fL
+	{ _RK_GLOBAL, "ssl.curves.list", _RK_C_STR,
```

Comment:
please use 8 whitespaces for indent (I know this is inconsistent in the code for historical reasons, but all new code should use this new style)

Reply:
no problem, fixed